src/video/haiku/SDL_bmodes.cc

```diff
@@ -148,6 +148,10 @@ int32 HAIKU_ColorSpaceToSDLPxFormat(uint32 colorspace)
     case B_RGB16_BIG:
         return SDL_PIXELFORMAT_RGB565;
         break;
+    case B_RGB24:
+    case B_RGB24_BIG:
+        return SDL_PIXELFORMAT_BGR24;
```

Comment:
I'm pretty sure B_RGB24 is SDL_PIXELFORMAT_BGR24 and B_RGB24_BIG is SDL_PIXELFORMAT_RGB24

Reply:
I was only able to test `B_RGB24`, so I'm not sure what pixel format `B_RGB24_BIG` actually requires in practice - so I used the exact same style as the existing cases

I can remove the untested B_RGB24_BIG case if that would be preferable

Reply:
Okay, well if it's wrong, at least it's consistently wrong with the other cases. :)